### ISSUE TITLE
Error after reloading post on frontend 

### ISSUE THREAD

**Issue by moritzebeling** [encoding-group/svelte-website#31]
1. Go on post in frontend
2. Edit + save that post in backend
3. Reload page in frontend

Results in the following:
<img width="1321" alt="Bildschirmfoto 2020-09-09 um 10 22 04" src="https://user-images.githubusercontent.com/39742729/92573614-50998800-f286-11ea-81a1-c657b89445a7.png">

**Comment by moritzebeling** [comment]
Dafür muss man nichts in Wordpress ändern. Es reicht, im Fontäne auf einen Post zu gehen und dann die Seite neu zu laden.

**Comment by reneweiser** [comment]
There is a [netlify function](https://www.netlify.com/products/functions/) that returns the following output everytime this happens:

```
1:07:19 AM: 2020-09-10T23:07:19.571Z	9a85fb3d-e980-4bf3-b23f-bc84a156b2ac	INFO	/archive/wip-magazine-4 - 76ms
1:07:19 AM: 2020-09-10T23:07:19.576Z	9a85fb3d-e980-4bf3-b23f-bc84a156b2ac	ERROR	Unhandled Promise Rejection 	{"errorType":"Runtime.UnhandledPromiseRejection","errorMessage":"TypeError: Cannot read property 'createElement' of undefined","reason":{},"promise":{},"stack":["Runtime.UnhandledPromiseRejection: TypeError: Cannot read property 'createElement' of undefined","    at process.<anonymous> (/var/runtime/index.js:35:15)","    at process.emit (events.js:315:20)","    at processPromiseRejections (internal/process/promises.js:209:33)","    at processTicksAndRejections (internal/process/task_queues.js:98:32)"]}
1:07:19 AM: [ERROR] [1599779239592] LAMBDA_RUNTIME Failed to post handler success response. Http response code: 403.
1:07:19 AM: Duration: 136.06 ms	Memory Usage: 128 MB	
1:07:19 AM: RequestId: 9a85fb3d-e980-4bf3-b23f-bc84a156b2ac Process exited before completing request

1:07:20 AM: 2020-09-10T23:07:20.721Z	e81c4aa8-0e4b-485b-86b9-d1c7b1f46a28	INFO	/favicon.ico - 283ms
1:07:20 AM: Duration: 295.38 ms	Memory Usage: 51 MB	
```

The actual error:
```json
{
  "errorType": "Runtime.UnhandledPromiseRejection",
  "errorMessage": "TypeError: Cannot read property 'createElement' of undefined",
  "reason": {},
  "promise": {},
  "stack": [
    "Runtime.UnhandledPromiseRejection: TypeError: Cannot read property 'createElement' of undefined",
    "    at process.<anonymous> (/var/runtime/index.js:35:15)",
    "    at process.emit (events.js:315:20)",
    "    at processPromiseRejections (internal/process/promises.js:209:33)",
    "    at processTicksAndRejections (internal/process/task_queues.js:98:32)"
  ]
}
```

According to this it has to do with some promise that got rejected and nothing is catching the exception. Not sure yet. Gonna dig a little more tomorrow.

**Comment by reneweiser** [comment]
It seems to work now (while writing this comment), but it could change at any time a commit triggers deployment of the site.

There is an error somewhere in the deployment process that causes ssr not to complete properly.

**Comment by reneweiser** [comment]
Should be fixed with c53cb4ea470a670152667648fb316b05e4cbcc66. Closing this.
